cmd/kapp/kapp.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2020 VMware, Inc.
+// Copyright 2024 The Carvel Authors.
```

Comment:
I updated all the copyright years to 2024 as I figured that would be required. If folks feel that is unnecessary or is not best practice/allowed I'll revert the copyright years and only change from `VMWare, Inc` --> `The Carvel Authors`

Reply:
May I know why do you feel it will be required to change year? As per my understanding we add the year as evidence of when the file has been written.

Reply:
I think it makes sense to switch to the 2024. @microwavables, @joaopapereira Do you have any thoughts/concerns with it?

Reply:
I am ok with the move to 2024